Zend/zend_compile.h

```diff
@@ -940,10 +940,12 @@ ZEND_API zend_string *zend_type_to_string(zend_type type);
 #define BP_VAR_UNSET		5
 
 #define ZEND_INTERNAL_FUNCTION		1
-#define ZEND_USER_FUNCTION			2
-#define ZEND_EVAL_CODE				4
+#define ZEND_FIBER_FUNCTION			2
```

Comment:
This is not okay, you'll want some other way to identify the function -- maybe via flag, or possibly by just comparing against the specific function (it's a unique one, right?)

Reply:
I need something to distinguish the stack frame I'm creating in `zend_fiber_execute` from other functions on the stack. What would you suggest? A flag on `zend_function.common.fn_flags` sounds fine too.

Reply:
I suppose I can have a `zend_is_fiber_function` that can compare the execute_data->func pointer to a specific one. That's simple.

Reply:
Here's a different approach: https://github.com/trowski/php-src/commit/e478035881fb845a0583eff45c0db70a3663f800